requirements/common.txt

```diff
@@ -52,3 +52,4 @@ python-geoip-geolite2
 pycountry
 pytz
 diff-match-patch
+google-compute-engine
```

Comment:
This is not related to the PR, travis fails without this. Remember the switch to the new travis images sometime ago in nextgen ?